.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,39 @@
+name: Feature request
+description: File a feature request.
+labels: ["enhancement"]
+body:
+  - type: markdown
+    attributes:
+      value: "Thanks for taking the time to fill out this feature request form!"

```

Comment:
Not pressing at the moment but we may want to have a discussion about feature requests actually become features as it relates to the PMC. cc: @clalancette